helm-charts/falcon-sensor/values.yaml

```diff
@@ -104,6 +104,9 @@ node:
     postDelete:
       enabled: true
 
+  # When enabled, Cloud data protection module is enabled
+  cdpRolesEnabled: true
```

Comment:
If data protection is not a separate sku, why are we adding this boolean? Permissions should be applied by default.